components/eam/src/physics/crm/samxx/crm_variance_transport.cpp

```diff
@@ -123,6 +127,8 @@ void VT_diagnose() {
       q_mean(k,icrm) = 0.0;
       t_vt(k,icrm) = 0.0;
       q_vt(k,icrm) = 0.0;
+      u_mean(k,icrm) = 0.0;
```

Comment:
You can move this line after q_mean = 0.0 so it is consistent to the rest of the code

Reply:
done.